package.json

```diff
@@ -27,6 +30,7 @@
     "@types/mocha": "^5.2.7",
     "aegir": "ipfs/aegir#feat/cosmiconfig",
     "chai": "^4.2.0",
+    "karma-mocha-webworker": "^1.3.0",
```

Comment:
You should not need this, karma has a weird issue with this module and fails to load it if the package doesnt come from npm. So when we release aegir properly this can be removed.

Reply:
oh, did not know that, failed locally so ive added it. Will remove once we update aegir version

Reply:
Even with released aegir it still says that it's missing